akka-docs/rst/java/cluster-metrics.rst

```diff
@@ -0,0 +1,159 @@
+
+.. _cluster_metrics_java:
+
+Cluster Metrics Extension
+=========================
+
+Introduction
+------------
+
+The member nodes of the cluster can collect system health metrics and publish that to other cluster nodes
+and to the registered subscribers on the system event bus with the help of Cluster Metrics Extension.
+
+Cluster metrics information is primarily used for load-balancing routers,
+and can also be used to implement advanced metrics-based node life cycles,
+such as "Node Let-it-crash" when CPU steal time becomes excessive.
+
+Cluster Metrics Extension is a separate akka module delivered in ``akka-cluster-metrics`` jar. 
+
+To enable usage of the extension you need to add the following dependency to your project: 
+::
+
+  <dependency>
+    <groupId>com.typesafe.akka</groupId>
+    <artifactId>akka-cluster-metrics_@binVersion@</artifactId>
+    <version>@version@</version>
+  </dependency>
+
+and add the following configuration stanza to your ``application.conf``
+::
+
+   akka.extensions = [ "akka.cluster.metrics.ClusterMetricsExtension" ]
+ 
```

Comment:
perhaps also mention

    # Disable legacy metrics in akka-cluster.
    akka.cluster.metrics.enabled=off